contributing.md

```diff
@@ -1,82 +1,88 @@
+
 ```
-git add <list of your changed files>
+git clone github.com/<githubusername>/t-route.git
 ```
-7. ... and commit them. As suggested in the example, a small commit with a precise comment, is often
-more useful that a massive commit with a detailed changelist buried in bullets.
+
+2. Set up the Git configuration parameters to specify your username and email address: 
+
+```
+git config --global user.name "Your GitHub username"
+git config --global user.email "Your email address"
 ```
-git commit -m ‘Make Brief comments to explain your Feature
 
-If you need additional lines for more detail, add them below a double space, like this.
--Bullets are useful for a large commit
--But usually, it’s better to just commit more often
--With a short one-line description of each little change’
-```
-8. In order to get any accumulated commits in GitHub on your fork, push the changes using `git push`
-9. After you git push all commits to your fork and you believe you are ready to post code to the main 
-repository, open GitHub and issue a pull request. (It will probably be a highlighted button at the top of the 
-page -- "New Pull Request").
-Of course, please test, document, style check (we use ['Black'][https://pypi.org/project/black/]), and prepare (as-needed) demonstration notebooks, etc. 
-More information about pull requests can be found [here][https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests]. 
-From that page:
-> "Pull requests let you tell others about changes you've pushed to a branch in a repository on GitHub. 
-Once a pull request is opened, you can discuss and review the potential changes with collaborators and 
-add follow-up commits before your
-changes are merged into the base branch."
-A pull request will allow someone else to look at your code with you to make sure that it is ready to share 
-with the world. Most of the time, someone who was involved with preparing the code can be the reviewer; for 
-major changes, it should be someone outside the core development team.
-9. **IMPORTANT** After you have issued a pull request the master upstream repository (NOAA-OWP) will have 
-been updated with your new code.
-It is important to make sure your fork is kept up-to-date with these new changes with the following commands. 
-The first command
-adds the OWP repository as an upstream remote that can be 'fetched' from to get the updates.
+3. Set up a remote connection to the newly forked repo. Call the remote "origin" with branch "master"
+
+```
+git remote add origin master https://github.com/<user.name>/t-route.git
+```
+
+4. If you wish to occasionally fetch and merge updates from the main t-route codebase, set the upstream remote accordingly. Use the `fetch` and `pull` command to occasionally retrieve updates from the main codebase and incorporate them into your local copy. 
```

Comment:
`fetch` and `pull` are redundant...a `git pull` is essentially `git fetch && git merge`.

Reply:
Thanks for catching this! I'll change ASAP.